sdk/include/opentelemetry/sdk/metrics/aggregation/histogram_aggregation.h

```diff
@@ -56,6 +71,31 @@ class DoubleHistogramAggregation : public Aggregation
   mutable HistogramPointData point_data_;
 };
 
+template <class T>
+void HistogramMerge(HistogramPointData &current,
+                    HistogramPointData &delta,
+                    HistogramPointData &merge)
+{
+  for (size_t i = 0; i < current.counts_.size(); i++)
```

Comment:
Are we sure all `counts_` have the same size?

Reply:
Yes it should be same for given instrument storage. We allocate it's size based on the size of bucket/boundary. For given instrument storage the bucket size will eventually be configured during View configuration ( as of now, we just use default bucket values from specs):

https://github.com/open-telemetry/opentelemetry-cpp/blob/75c2a8f7a6bf81d9799e76804473609cc236b7be/sdk/src/metrics/aggregation/histogram_aggregation.cc#L17-L20